tests/qam-updinstall/update_install.pm

```diff
@@ -145,7 +145,11 @@ sub run {
 
     # Install released version of installable binaries.
     if (scalar(keys %installable)) {
-        zypper_call("in -l " . join(' ', keys %installable), exitcode => [0, 102, 103], log => 'prepare.log', timeout => 1500);
+        my $ret = zypper_call("in -l " . join(' ', keys %installable) . '|tee zypper.log', exitcode => [0, 8, 102, 103], log => 'prepare.log', timeout => 1500);
```

Comment:
Why is there `|tee zypper.log` and then also `log => 'prepare.log'`? Is that correct?

Reply:
I will fix that. It's doing the same.